CoRIM locator map as a hint (#578)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

### Locator Map {#sec-corim-locator-map}

> The locator map acts as a hint that contains pointers to repositories where dependent manifests,

Reviewer: ```suggestion
The locator map contains pointers to repositories where dependent manifests,
```

The locator map contains pointers to repositories where dependent manifests,
certificates, or other relevant information can be retrieved by the Verifier.
The contents of the locator map are purely advisory.

> Verifiers are not required to follow these links or use the information retrieved from them.

Reviewer 2: ```suggestion
Verifiers are not required to follow these URLs or use the information retrieved from corresponding resources.
```

> Verifiers are not required to follow these URLs or use the information retrieved from corresponding resources.

Reviewer 2: ```suggestion
Verifiers are not required to follow these URLs or use the information retrieved from the corresponding resources.
```
